### ISSUE TITLE
Converters are useful in our model.

### ISSUE THREAD

**Issue by bulky-kohsakunishida** [NVIDIA-AI-IOT/torch2trt#441]
We implemented a Convolutional AutoEncoder model (image size 512x512, activation function Mish, and loss function ms-ssim) using pytorch for product inspection for a small manufacturing company. This model takes 15 seconds to infer on the jetson Xavier NX, while the torch2trt-converted model took only 1.5 seconds.

This speed is very useful as it is deemed practical enough.

Thanks.
